Gui/opensim/CMakeLists.txt

```diff
@@ -16,6 +16,22 @@ endif()
 
 find_package(Ant REQUIRED)
 
+# http://xit0.org/2013/04/cmake-use-git-branch-and-commit-details-in-project/
+# Get the latest abbreviated commit hash of the working branch
+execute_process(
+  COMMAND git log -1 --format=%h
+  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+  OUTPUT_VARIABLE GIT_COMMIT_HASH
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+string(TIMESTAMP timestamp)
+set (BUILD_ID_RAW ${timestamp}.${GIT_COMMIT_HASH})
+# Replace illegal char in Path ":" with "."
+string(REPLACE ":" "." BUILD_ID ${BUILD_ID_RAW})
+message(${BUILD_ID})
```

Comment:
Please make this message more descriptive; perhaps `message("Build ID: ${BUILD_ID}")`.